lib/fedora_migrate/repository_migrator.rb

```diff
@@ -67,9 +71,8 @@ def qualifying_object object
       return object if name.match(namespace)
     end
 
-    def find_or_create_report source
```

Comment:
I did that to try and distinguish between the report that the migrator is building, i.e. `@report`, and the individual reports for each object that gets migrated. I started getting confused myself between `Report`, `report` and `@report`!